### ISSUE TITLE
Add --extension to Piranha Micro page

### ISSUE THREAD

**Issue by manorrock** [piranhacloud/piranha#944]

**Comment by Thihup** [comment]
We don't have the --extension flag in Piranha Micro, for now. We only have the _piranha.extensions_ property. Maybe create an issue to add the --extension flag?

**Comment by manorrock** [comment]
Blocked by #945

**Comment by manorrock** [comment]
Closing as won't fix for now as we do have the piranha.extensions property
